### ISSUE TITLE
Condition for number of buffs on group. 

### ISSUE THREAD

**Issue by robinsone** [ascott18/TellMeWhen#1989]
**Is your feature request related to a problem? Please describe.**
I think it would be a good idea to implement a feature that would easily allow us to track the number of buffs that are currently present in a party or raid. If we could use it in a condition we could easily make decision on that information. 

**Describe the solution you'd like**
Implement a new buff/debuff condition that counts the number of that buff/debuff available on a party or raid. An example of the scenario I had in mind. Currently in the process of making a rotation helper for Mistweaver monks. There's a point where if there are more than 6 renewing mists on injured allies in the raid, vivify should be used instead of essence font during a high damage period. It would be great if I could use a reactive spell icon with the condition that the party/raid has more than 6 renewing mists applied to members. 

**Describe alternatives you've considered**
I've tried to use a "buff - number of" condition to get this effect but it doesn't seem to work. I've also tried setting up a buff/debuff unit condition and then make it a group controller. Then use a counter on shown/hide to increase/decrease the active renewing mists count. However, it's not working as expected with the spell reactivity. 

**Additional context**
Weakaura has a easy way to do this with their smart group trigger. However, I much prefer TMW for it's ease of use and configuration for rotation helpers / spell HUDs. 

Thank you for considering this feature. And as always great job on this addon.

**Comment by arieh** [comment]
This can be done today with TMW using a LUA condition, though it's not as ergonomic or efficient as having it as a core feature.

**Comment by robinsone** [comment]
Yeah I'm not well versed in LUA. I would make an argument that it would be wonderful as a core feature for healers.

**Comment by arieh** [comment]
@ascott18 - initially I was thinking this was too niche for TMW since the addon doesnt have this kind of one-to-many conditions, however when playing with it a little I did find a very common use case for the general functionality of tracking a buff's presence across the group - checking if the enitre group has a certain raid buff (battle-shout etc).

My general concern here is that it's hard to optimize around.

@robinsone - I've added both `Group Buff Count` and `All Group has Buff` conditions to TellMeWhen Script Tools if you want to play with it.
@ascott18  - [the code is here](https://github.com/arieh/TellMeWhen_Script_Tools/blob/master/modules/GroupBuffCount.lua) - if you think this is something worthwhile for TMW let me know and I'll work on a PR
